core/amber/src/main/scala/edu/uci/ics/amber/engine/operators/SinkOpExecConfig.scala

```diff
@@ -0,0 +1,5 @@
+package edu.uci.ics.amber.engine.operators
```

Comment:
why do we need this file?

Reply:
because we may create more sink operators.